### PR TITLE
Ignore NotFoundHttpExceptions by default, make the list of ignored

### PR DESCRIPTION
**Pull request by Jalle19**
exceptions configurable through the constructor.

@kryysler does this look okay?

### CODE REVIEW

File: src/NewRelicExceptionHandler.php [hunk 1]
```diff
@@ -13,11 +14,33 @@ class NewRelicExceptionHandler implements ExceptionHandler
...
+    }
+
+
+    /**
      * @inheritdoc
      */
     public function report(Exception $e)
     {
-        if (extension_loaded('newrelic')) {
+        if (extension_loaded('newrelic') && in_array(get_class($e), $this->ignoredExceptions)) {
```
Comment (kryysler) [inline src/NewRelicExceptionHandler.php]: Shouldn't it be the other way around, e.g. "not in array"?

Comment (Jalle19) [reply src/NewRelicExceptionHandler.php]: Yeah, noticed that too :D

**Comment by Jalle19** [comment]
Trying to come up with a way to test this, kind of hard when it relies on a function from an extension.

**Comment by Jalle19** [comment]
Merged manually

**Pull request closed without merging**